parachain/primitives/core/src/outbound.rs

```diff
@@ -6,6 +6,13 @@ use sp_std::{borrow::ToOwned, vec, vec::Vec};
 
 pub type MessageHash = H256;
 
+#[derive(Copy, Clone, Encode, Decode, PartialEq, Eq, RuntimeDebug, TypeInfo)]
+pub enum Priority {
+	Normal,
+	High,
+	Emergency,
```

Comment:
I'd like us to make conservative, incremental changes. So if we are not actually implementing `Emergency` priority, then I'd rather we not include this variant.

```suggestion
```